bucket/vortexdm.json

```
@@ -0,0 +1,34 @@
+{
+    "version": "2022.1.0",
+    "description": "Multi-connections download manager",
+    "homepage": "https://github.com/Sixline/VDM",
+    "license": "LGPL-3.0-only",
+    "url": "https://github.com/Sixline/VDM/releases/download/2022.1.0/VortexDM-2022.1.0-win64.zip",
+    "hash": "dd491bce137ce392f16525a4868eadb77b52d30719424f41231918679db04a22",
+    "extract_dir": "VortexDM",
+    "pre_install": [
+        "'downloads.dat', 'thumbnails.dat', 'user_themes.cfg' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" -ItemType File | Out-Null }",
```

```suggestion
        "    if (!(Test-Path \"$persist_dir\\$_\")) { New-Item \"$dir\\$_\" | Out-Null }",
```
`-ItemType File` is not needed because *New-Item* create it as a file by default.